flowey/flowey_hvlite/src/pipelines/checkin_gates.rs

```diff
@@ -709,6 +723,30 @@ impl IntoPipeline for CheckinGatesCli {
                 );
 
             all_jobs.push(job.finish());
+
+            if arch == CommonArch::X86_64 && matches!(config, PipelineConfig::Pr) {
```

Comment:
we're publishing aarch64 baselines, but not actually checking them. is that expected (i.e: something we'll enable in a follow up-PR), or is the x86 gate leftover from an earlier iter?